docs/contributing/running-and-writing-acceptance-tests.md

```diff
@@ -1161,44 +1161,61 @@ func testSweepExampleThings(region string) error {
   }
 
   conn := client.(*AWSClient).exampleconn
+  sweepResources := make([]*testSweepResource, 0)
+  var errors *multierror.Error
```

Comment:
Ack, I should have noticed this before, but `errors` matches the [standard library `errors` package name](https://pkg.go.dev/errors) with error handling functionality. Probably not worth changing now, but 😢  for potential confusion.

Reply:
We're still in the bud stage with this so I rather nip it.